terraform/cloud-platform-components/prometheus.tf

```diff
@@ -0,0 +1,74 @@
+resource "helm_repository" "coreos" {
+  name = "coreos"
+  url  = "https://s3-eu-west-1.amazonaws.com/coreos-charts/stable/"
+}
+
+resource "kubernetes_storage_class" "prometheus" {
+  metadata {
+    name = "prometheus-storage"
+  }
+
+  storage_provisioner = "kubernetes.io/aws-ebs"
+
+  parameters {
+    type = "gp2"
+  }
+}
+
+resource "helm_release" "prometheus_operator" {
+  name          = "prometheus-operator"
+  chart         = "coreos/prometheus-operator"
+  namespace     = "monitoring-terraform"
+  recreate_pods = "true"
+
+  depends_on = [
+    "null_resource.deploy",
+    "helm_repository.coreos",
+  ]
+}
+
+resource "helm_release" "kube_prometheus" {
+  name          = "kube-prometheus"
+  chart         = "coreos/kube-prometheus"
+  namespace     = "monitoring-terraform"
+  recreate_pods = "true"
+
+  values = [
+    "${file("../../helm-charts/prometheus-operator/kube-prometheus/values.yaml")}",
+  ]
+
+  set {
+    name  = "grafana.extraVars[0].value"
+    value = "https://grafana.apps.${data.terraform_remote_state.cluster.cluster_domain_name}"
+  }
+
```

Comment:
From the README:
```
  --set-string grafana.adminUser=$(head -c 16 /dev/urandom | xxd -p) \
  --set-string grafana.adminPassword=$(head -c 16 /dev/urandom | xxd -p)
```
Although we agreed we won't manage the extra bits yet, we still need to override these values. `hex` from [`random_id`](https://www.terraform.io/docs/providers/random/r/id.html) should be equivalent.

Reply:
Okay. This has been done under b88057b.